e2e/stack_monitoring_checks_test.go

```diff
@@ -0,0 +1,406 @@
+package e2e
```

Comment:
Running `go test -v ./...` from the e2e dir, or using VSCode to debug the tests helped me to debug and improve the helper functions